cmd/oras/internal/option/confirmation.go

```diff
@@ -26,18 +26,18 @@ import (
 
 // Confirmation option struct.
 type Confirmation struct {
-	Confirmed bool
+	Force bool
 }
 
 // ApplyFlags applies flags to a command flag set.
 func (opts *Confirmation) ApplyFlags(fs *pflag.FlagSet) {
-	fs.BoolVarP(&opts.Confirmed, "force", "f", false, "do not prompt for confirmation")
+	fs.BoolVarP(&opts.Force, "force", "f", false, "do not prompt for confirmation")
```

Comment:
nit: Should we consider `ignore non-existent references, never prompt` over `do not prompt for confirmation`? /cc @FeynmanZhou

Reply:
Looks good. Also nonexistent [is a word](https://www.merriam-webster.com/dictionary/nonexistent) and is already used by `rm` manual.

Reply:
Agree with @shizhMSFT 's suggestion. I see @qweeah has incorporated it into the confirmation.